testsuite/libra_fuzzer/fuzz/Cargo.lock

```diff
@@ -0,0 +1,3832 @@
+# This file is automatically @generated by Cargo.
```

Comment:
@sunshowers  should I not include this file?

Reply:
Yeah not at the moment, I want to get around to doing this properly.

Reply:
removed!